commit 1ff5f3d2a0fcd48add9462567c30fe0e14585fb4
Author: Matt Whitlock <whitslack@users.noreply.github.com>
Date:   Wed Jun 9 10:25:18 2021 -0400

    configure.ac: fix SYSCONFDEFDIR default value
    
    configure.ac was using AC_ARG_WITH incorrectly, yielding a generated configure script like:
    
        # Check whether --with-sysconfdefdir was given.
        if test "${with_sysconfdefdir+set}" = set; then :
          withval=$with_sysconfdefdir; SYSCONFDEFDIR=$withval
        else
          "/etc/sysconfig"
        fi
    
    This commit fixes the default case so that the SYSCONFDEFDIR variable is assigned the value "/etc/sysconfig" rather than trying to execute "/etc/sysconfig" as a command.
    
    Signed-off-by: Mauro Carvalho Chehab <mchehab+huawei@kernel.org>

diff --git a/configure.ac b/configure.ac
index f7d1947..33b81fe 100644
--- a/configure.ac
+++ b/configure.ac
@@ -172,7 +172,7 @@ AC_SUBST([RASSTATEDIR])
 AC_ARG_WITH(sysconfdefdir,
     AC_HELP_STRING([--with-sysconfdefdir=DIR], [rasdaemon environment file dir]),
     [SYSCONFDEFDIR=$withval],
-    ["/etc/sysconfig"])
+    [SYSCONFDEFDIR=/etc/sysconfig])
 AC_SUBST([SYSCONFDEFDIR])
 
 AC_DEFINE([RAS_DB_FNAME], ["ras-mc_event.db"], [ras events database])
